apps/web-giddh/src/app/contact/payment-aside/payment-aside.component.html

```diff
@@ -107,15 +106,10 @@ <h4 class="modal-title pull-left">Payment</h4>
                                 </div>
                             </div>
                             <div class="amount-remark-section">
-                                <p class="text-gray due-amount hidden-md hidden-lg hidden-sm mb-2">
-                                    Total Due:{{item.closingBalance.amount}}
-                                    <span class="red-text cursor-pointer pull-right mb-1" (click)="removeSelectedAccount(item); removeTransactionsDetailsForm(entryIdx)"><i
-                                            class="icon-trash mr-05"></i>Remove</span>
-                                </p>
                                 <div class="d-flex">
                                     <div class="form-group pr-15">
                                         <label>Amount<span class="text-danger">*</span></label>
-                                        <input class="form-control" name="closingBalanceAmount{{entryIdx}}" [disabled]="isPayClicked" required [(ngModel)]="item.closingBalance.amount" type="number">
+                                        <input class="form-control" name="closingBalanceAmount{{entryIdx}}" [(ngModel)]="item.closingBalance.amount" decimalDigitsDirective type="text" required [disabled]="isPayClicked">
```

Comment:
Required attribute should be conditionally true here since field can be disabled

Reply:
No need to do it

Reply:
Why?

Reply:
Done